# Hi-C

HiNT (Hi-C for copy Number variation and Translocation detection), a computational method to detect CNVs and Translocations from Hi-C data. 
https://github.com/parklab/HiNT
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7087379/
https://www.ncbi.nlm.nih.gov/sra?term=SRX950725


Identification and utilization of copy number information for correcting Hi-C contact map of cancer cell lines
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7648276/

Effective normalization for copy number variation in Hi-C data
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6127909/


Hi-C as a tool for precise detection and characterisation of chromosomal rearrangements and copy number variation in human tumours
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5488307/

Identification of copy number variations and translocations in cancer cells from Hi-C data
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8210825/

hicancer: accurate and complete cancer genome phasing with Hi-C reads
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7987978/

A computational strategy to adjust for copy number in tumor Hi-C data
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6078171/


Multiplex Chromatin Interactions with Single Molecule Precision
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7001875/

Hi ChIP
https://link.springer.com/protocol/10.1007%2F978-1-0716-1597-3_10


Capture Hi-C identifies putative target genes at 33 breast cancer risk loci
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5847529/


Computational Chromosome Conformation Capture by Correlation of ChIP-seq at CTCF motifs
https://github.com/ibn-salem/sevenC   MUTLAKA BAK!!!


